core/src/test/java/com/google/instrumentation/stats/StatsContextFactoryTest.java

```diff
@@ -24,8 +24,14 @@
  */
 @RunWith(JUnit4.class)
 public class StatsContextFactoryTest {
-  @Test(expected = IOException.class)
+  @Test
```

Comment:
Makes sense, though we should document that somewhere.

This test should then be updated to check that deserializing an empty byte array returns the default stats context.